pkg/commands/apply/apply.go

```diff
@@ -358,12 +361,17 @@ func (c *ApplyCommand) createStartCommentForActions(ctx context.Context) (*githu
 
 	c.Outf("Creating start comment")
 
+	destroyText := ""
+	if c.isDestroy {
+		destroyText = DestroyCommentText
+	}
+
```

Comment:
I see this same pattern repeated a number of times (thing, if destroy, thing = somethingelse). I've usually found that's a hint that using an interface (or separate command) is actually a better option.

Reply:
I was planning on creating a reporter interface, so this info can be passed of to create different strategies to create output, ie github, stdout, some other thing. Will resolve this in a follow up.